scipy/signal/tests/test_waveforms.py

```diff
@@ -81,6 +86,11 @@ def test_quadratic_at_zero2(self):
                             vertex_zero=False)
         assert_almost_equal(w, 1.0)
 
+    def test_quadratic_complex_at_zero(self):
```

Comment:
We would need to test more known values to ensure the function behaves as expected and there are no corner cases it cannot handle.

Reply:
Do you have any suggestions? 

PS. Sorry, this fell through the cracks. If you can guide me through some example scenarios, I can write them.

Reply:
Added one more test. I'll think about what else I can add.